h/static/images/icons/logo.svg

```diff
@@ -4,7 +4,9 @@
     <title>Rectangle 2 Copy 17</title>
     <desc>Created with Sketch.</desc>
     <defs></defs>
+    <rect fill="#ffffff" stroke="none" width="17.14407" height="16.046612"
+          x="3.8855932" y="3.9449153" />
```

Comment:
Alternatively I guess you could put a white h on top of the speech bubble shape instead of having an h-shaped hole in the speech bubble. That would be harder to do from what we have now but maybe I should do it...